src/api/forecastAPI.ts

```diff
@@ -39,3 +40,36 @@ export async function getNoonForecasts(stationCodes: number[]): Promise<Forecast
     throw err.toString()
   }
 }
+
+export interface ForecastSummary {
+  datetime: string
+  tmp_min: number
+  tmp_max: number
+  rh_min: number
+  rh_max: number
+}
+
+// List of noon forecast summaries for each datetime with station info
+export interface ForecastSummariesForStation {
+  station: Station
+  values: ForecastSummary[]
+}
+
+export interface ForecastSummariesResponse {
+  summaries: ForecastSummariesForStation[]
+}
+
+export async function getForecastSummaries(
+  stationCodes: number[]
+): Promise<ForecastSummariesForStation[]> {
+  const url = `/noon_forecasts/summaries/`
+  try {
+    const { data } = await axios.post<ForecastSummariesResponse>(url, {
+      stations: stationCodes
+    })
+
+    return data.summaries
+  } catch (err) {
+    throw err.toString()
```

Comment:
What is the purpose here? Why catch an exception, just to re-throw a string version of it?

Reply:
I have to throw an exception like this so that another wrapper of try & catch can catch it, and eventually, this error message gets dispatched in Redux actions.

Reply:
As an example, take a look at `fetchForecastSummaries` method in `forecastSummariesSlice.ts`